chapters/logging.rst

```diff
@@ -0,0 +1,242 @@
+==============================
+Logging configuration in Jina
+==============================
+
+.. meta::
+   :description: A guide on logging configuration in Jina
+   :keywords: Jina, logging
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+-------------------
+
+In order to better understand, monitor, and debug the running and lifetime of Jina's peas, pods, and flow, Jina offers logging to help log the messages.
```

Comment:
In order to monitor and debug the 

The understanding is not needed :)

Reply:
done